.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+#
+version: 2
+jobs:
+    build:
+        docker:
+            # specify the version you desire here
+            - image: circleci/node:12.14.1
+              environment: 
+                TEST_DB_USER: my_user
+                TEST_DB_NAME: my_test_db
+            #   environment:
+            #     TEST_DATABASE_URL: postgresql://root@localhost:8080/volunteersngtest
+            - image: postgres:9.6.2-alpine
+              environment:
+                POSTGRES_USER: my_user
+                POSTGRES_DB: my_test_db
+
+            # Specify service dependencies here if necessary
```

Comment:
Take out all the unnecessary comments in this file

Reply:
Okay